Source/WebCore/page/unified-text-replacement/UnifiedTextReplacementTypes.h

```diff
@@ -25,33 +25,89 @@
 
 #pragma once
 
-#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+#if PLATFORM(COCOA)
```

Comment:
Can you explain this change? `ENABLE` is generally preferred over platform conditionals.